CONTRIBUTING.md

```diff
@@ -179,51 +205,36 @@ Please ensure that all React UI components contributed meet the following guidel
 
 Adhering to the following process is the best way to get your work included in the project:
 
-1. [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork, and configure the remotes:
-
-```sh
-# Clone your fork of the repo into the current directory
-git clone https://github.com/<your-username>/patternfly-react.git
-# Navigate to the newly cloned directory
-cd patternfly-react
-# Assign the original repo to a remote called "upstream"
-git remote add upstream https://github.com/patternfly/patternfly-react.git
-# Fetch the code and branches from remote repo "upstream"
-git fetch upstream
-```
+1. **Fork and set up the repository** (assumes basic git knowledge)
```

Comment:
Maybe doesn't need to state the `assumes basic git knowledge`. We can maybe put a sentence about that up in the top of this section, and /or maybe we can point to git's docs about forking?